src/theme/components.js

```diff
@@ -98,7 +98,7 @@ const components = {
           overflow: 'hidden',
           textOverflow: 'ellipsis',
           whiteSpace: 'nowrap',
-          maxWidth: '450px',
+          maxWidth: '300px',
```

Comment:
- [ ] The issue of inconsistent width is still there and it cannot be fixed by reducing the `maxWidth` of the commit-message class. 



At desktop width             | 
:-------------------------:| 
![image](https://user-images.githubusercontent.com/60618877/225712423-ec1ead44-1d62-4402-afe6-3e79638968ac.png)

The idea is that the content should wrap when reaching the column width shown with red line          | 
:-------------------------:| 
![image](https://user-images.githubusercontent.com/60618877/225716499-467237a3-322d-4687-b18a-db50b0788212.png)


- Let's figure out how widths for these columns can be constrained despite the size of the content they contain, the goal being, the content within the columns should wrap to the next line when the content reaches the set width of the column.

- To ensure the widths are responsive to different screen sizes it is a good practice to provide the values in quantities of percentages/em/rem and not pixels (px).